iaso/api/mobile/reports.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework.pagination import LimitOffsetPagination
+
+from iaso.api.common import ModelViewSet
+from django_filters.rest_framework import DjangoFilterBackend  # type: ignore
+from rest_framework import filters, permissions
+from rest_framework import serializers
+from iaso.models import Report
+
+
+class MobileReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Report
+        fields = ["name", "url", "version_id", "version_name", "created_at", "updated_at"]
+
+    version_name = serializers.SerializerMethodField()
+    version_id = serializers.SerializerMethodField()
+    url = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_version_name(obj: Report):
+        return obj.published_version.name
+
+    @staticmethod
+    def get_url(obj: Report):
+        return obj.published_version.file.url
+
+    @staticmethod
+    def get_version_id(obj: Report):
+        return obj.published_version.id
```

Comment:
these can be replaced with Field using the source= parameter but it's not blocking. I can show you next time.